### PR TITLE
[FIX] synchronize.py: Delete the repeated repository for each project

### PR DESCRIPTION
**Pull request by JesusZapata**
When the script `maintainer-quality-tools/travis_weblate.py` run and news components are created the repository source is bad because the repository is the base repository and this causes the creation of another repository.

In Weblate should be exist only one repository per project, the others projects in the fields repository should be contains the `weblate://` repository

For example:

The follow project on Weblate has a two repository core, it is bad because should be exist only one repository per project

image

And after run the `SynRunbotWeblate/synchronize.py` with this change the trouble was fixed

image

image

### CODE REVIEW

File: synchronize.py [hunk 1]
```diff
@@ -114,9 +129,65 @@ def find_or_create_project(self, project):
...
+                        'slug': component['slug'],
+                        'branch': component['branch'],
+                        'repo': base_repo}])
+        logger.info('WeblateAPI._fix_bad_repository > '
+                    'Fix bad repository url git %s '
+                    'base_component %s and component %s '
+                    '(cmd=%s)', project['slug'], base_component['slug'],
+                    component['slug'], cmd)
+        try:
+            print subprocess.check_output(cmd)
```
Comment (moylop260) [inline synchronize.py]: What about `logger.info`?

**Comment by moylop260** [comment]
We don't have a way to test it
Is possible upload a video testing it?

**Comment by JesusZapata** [comment]
@moylop260 
Sorry I forget as recreated this fail but I recording the follow video assuming that trouble previous happened
The first trouble was https://github.com/Vauxoo/maintainer-quality-tools/pull/247 this escenario happened on real weblate server with real data

https://www.youtube.com/watch?v=BacsyV3Gjyg

**Comment by JesusZapata** [comment]
@moylop260 

In the follow video you can see one real problem  https://youtu.be/CdwYfvjMg6c

When the weblate contains many source repository for one repository the `MQT` show the follow error `many repository for this project`

**The real cause of that problem I do not know**

Below we can see the data related with this problem, in this JSON data we can see 3 components, in a normal case all components should be has empty value on `git_export` and only one component set this value. 

But in this case 2 components has `git_export` and this is problem, because each `git_export` equals to a git repository, and each project should be have exactly one repository


```json
{
    "count": 3,
    "next": null,
    "previous": null,
    "results": [
        {
            "name": "account_move_template",
            "slug": "account_move_template",
            "vcs": null,
            "repo": null,
            "git_export": "",
            "branch": null,
            "filemask": null,
            "template": "",
            "new_base": "",
            "file_format": "auto",
            "license": "",
            "license_url": "",
            "web_url": "http://example.com/projects/github_com_Vauxoo_tanner_11_0/account_move_template/",
            "url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/account_move_template/",
            "repository_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/account_move_template/repository/",
            "translations_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/account_move_template/translations/",
            "statistics_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/account_move_template/statistics/",
            "lock_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/account_move_template/lock/",
            "changes_list_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/account_move_template/changes/"
        },
        {
            "name": "tanner",
            "slug": "tanner",
            "vcs": null,
            "repo": null,
            "git_export": "http://example.com/git/github_com_Vauxoo_tanner_11_0/tanner/",
            "branch": null,
            "filemask": null,
            "template": "",
            "new_base": "",
            "file_format": "auto",
            "license": "",
            "license_url": "",
            "web_url": "http://example.com/projects/github_com_Vauxoo_tanner_11_0/tanner/",
            "url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner/",
            "repository_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner/repository/",
            "translations_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner/translations/",
            "statistics_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner/statistics/",
            "lock_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner/lock/",
            "changes_list_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner/changes/"
        },
        {
            "name": "tanner_sale_margin",
            "slug": "tanner_sale_margin",
            "vcs": null,
            "repo": null,
            "git_export": "http://example.com/git/github_com_Vauxoo_tanner_11_0/tanner_sale_margin/",
            "branch": null,
            "filemask": null,
            "template": "",
            "new_base": "",
            "file_format": "auto",
            "license": "",
            "license_url": "",
            "web_url": "http://example.com/projects/github_com_Vauxoo_tanner_11_0/tanner_sale_margin/",
            "url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner_sale_margin/",
            "repository_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner_sale_margin/repository/",
            "translations_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner_sale_margin/translations/",
            "statistics_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner_sale_margin/statistics/",
            "lock_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner_sale_margin/lock/",
            "changes_list_url": "http://weblate.vauxoo.com/api/components/github_com_Vauxoo_tanner_11_0/tanner_sale_margin/changes/"
        }
    ]
}
```

This change solves this problem

**Comment by JesusZapata** [comment]
@moylop260 
This pull request not have a conflict

**Pull request merged**